sqldelight-gradle-plugin/src/main/kotlin/app/cash/sqldelight/gradle/SqlDelightDatabase.kt

```diff
@@ -32,6 +33,7 @@ abstract class SqlDelightDatabase @Inject constructor(
   abstract val migrationOutputDirectory: DirectoryProperty
   val migrationOutputFileFormat: Property<String> = project.objects.property(String::class.java).convention(".sql")
   val generateAsync: Property<Boolean> = project.objects.property(Boolean::class.java).convention(false)
+  val connectionProperties: MapProperty<String, String> = project.objects.mapProperty(String::class.java, String::class.java)
```

Comment:
I don't think, adding it to the database is useful to the most users. It is very specific to your case, so adding it to the sqldelightTasks would be enough.

Reply:
Good point, removed!